### DISCUSSION TITLE
Meeting 2024-08-27

### DISCUSSION THREAD

**Discussion by adamgreig** [rust-embedded/wg#789] [General]
This is the agenda for the next meeting. Please add anything you'd like to discuss below!

* Meetings: Tuesday 8pm Europe/Berlin time
* [Join the Matrix chat](https://matrix.to/#/#rust-embedded:matrix.org)
* [Nominated issues](https://github.com/search?q=org%3Arust-embedded+label%3Anominated+is%3Aopen&type=Issues)
* [IRC/Matrix logs](https://libera.irclog.whitequark.org/rust-embedded/2024-08-27)

### Announcements

* Our survey is live! https://www.surveyhero.com/c/uenp3ydt
* critical-section 1.1.3 released with new `usize` restore state option
*

**Comment by rmsyn** [comment]
From the triage meeting a couple weeks back, https://github.com/rust-embedded/wg/issues/361 appears resolved.

Could probably use more sets of eyes to confirm resolution.

**Comment by adamgreig** [comment]
Updated embedded-io Read trait documentation to clarify when calls will block: https://github.com/rust-embedded/embedded-hal/pull/625

**Comment by adamgreig** [comment]
We discussed the recent activity on https://github.com/rust-embedded/wg/pull/387. Given the RFC itself is from 2019 and the general conversion around the svd2rust API and how to avoid references to MMIO has moved on a long way since then, I think we can close the PR. However, it became a useful issue to discuss the general ref-to-MMIO issue which we don't otherwise have a place for, so I'll open a new issue for that.

**Comment by adamgreig** [comment]
The updates on https://github.com/rust-lang/rust/issues/53639 were also mentioned but they can be discussed in https://matrix.to/#/#wg-embedded-static-mut:matrix.org instead.
